.github/workflows/workflow.yml

```
@@ -0,0 +1,64 @@
+name: codeRabbit action
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+    types: [opened]
+    branches-ignore:
+      - main
+    pull_request_review_comment:
```

actionlint (1.7.7)

unexpected key "pull_request_review_comment" for "pull_request" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)